### PR TITLE
Add VERBOSE option and example and MVCC prop.

### PR DESCRIPTION
**Pull request by stbof**
DOC-1727.

### CODE REVIEW

File: netlify/vale/vocab.txt [hunk 1]
```diff
@@ -203,6 +203,7 @@ hueypark
 hyperthread
 hyperthreads
 img
+include_cached
```
Comment (stbof) [inline netlify/vale/vocab.txt:206]:
```suggestion
```

Comment (stbof) [reply netlify/vale/vocab.txt:206]: Fixed in Vale.

File: v22.1/explain-analyze.md [hunk 2]
```diff
@@ -76,6 +76,8 @@ KV rows read | During scans, the number of rows in the [storage layer](architect
 KV bytes read | During scans, the amount of data read from the [storage layer](architecture/storage-layer.html) during this phase of the statement.
 estimated max memory allocated  | The estimated maximum allocated memory for a statement.
 estimated max sql temp disk usage | The estimated maximum temporary disk usage for a statement.
+MVCC step count (ext/int) | The number of times that the underlying storage iterator stepped forward during the work to serve the operator's reads, including stepping over MVCC keys that could not be used in the scan.
```
Comment (mgartner) [inline v22.1/explain-analyze.md:79]: Should we link MVCC (or MVCC keys) to https://www.cockroachlabs.com/docs/stable/architecture/storage-layer.html#mvcc?

**Review by mgartner — APPROVED** [review]
LGTM

**Review by taroface — APPROVED** [review]
LGTM

**Pull request merged**